### PR TITLE
Bump Smallrye Reactive Messaging version to 3.22.1

### PR DESCRIPTION
**Pull request by ozangunalp**
https://github.com/smallrye/smallrye-reactive-messaging/releases/tag/3.22.1

### CODE REVIEW

**Comment by ozangunalp** [comment]
@jponge fyi

File: extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/WiringHelper.java [hunk 1]
```diff
@@ -122,7 +123,8 @@ static boolean isInboundConnector(ClassInfo ci) {
      */
     static boolean isOutboundConnector(ClassInfo ci) {
         // TODO Add the internal interface support
-        return ci.interfaceNames().contains(ReactiveMessagingDotNames.OUTGOING_CONNECTOR_FACTORY);
+        return ci.interfaceNames().contains(ReactiveMessagingDotNames.OUTGOING_CONNECTOR_FACTORY)
```
Comment (cescoffier) [inline extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/WiringHelper.java:126]: I think the todo can be removed

File: extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/WiringHelper.java [hunk 2]
```diff
@@ -111,7 +111,8 @@ static boolean isChannelEnabled(ChannelDirection direction, String channel) {
      */
     static boolean isInboundConnector(ClassInfo ci) {
         // TODO Add the internal interface support
-        return ci.interfaceNames().contains(ReactiveMessagingDotNames.INCOMING_CONNECTOR_FACTORY);
+        return ci.interfaceNames().contains(ReactiveMessagingDotNames.INCOMING_CONNECTOR_FACTORY)
```
Comment (cescoffier) [inline extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/WiringHelper.java:114]: I think the todo can be removed

**Review by cescoffier — APPROVED** [review]
LGTM - just two todos that can be removed.

**Pull request merged**